lib/Conf.mli

```diff
@@ -107,7 +107,7 @@ type opts =
         (** Check whether the formatted output exceeds the margin. *) }
 (** Options changing the tool's behavior *)
 
-val action : unit -> action * opts
+val action : unit -> action Cmdliner.Term.result * opts
```

Comment:
This should be `(action * opts) Cmdliner.Term.result`. Eg. constructing `opts` in `validate` in lib/Conf.ml

Reply:
done